playground_server_design.md

```diff
@@ -0,0 +1,130 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server expose an REST API service which enable users to share
```

Comment:
If this is an implementation just for DSW to work, I'd recommend you put this doc in some internal places and explain how and why we should do this.

For public uses, maybe we only need the cloud playground and the VM box or some other approach that users can download and setup everything on his laptop or setup a cluster for his company. And, this setup method described in this doc gives less control over the SQLFlow server nor the database, it seems the same using this setup than the cloud playground.

Reply:
Emm, agreed.